src/Engine/Component/GeometryComponent.cpp

```diff
@@ -73,33 +73,32 @@ void TriangleMeshComponent::generateTriangleMesh( const Ra::Core::Asset::Geometr
     std::string name( m_name );
     name.append( "_" + m_contentName );
 
-    std::string meshName = name;
-    meshName.append( "_Mesh" );
+    std::string meshName = m_contentName;
```

Comment:
This change is not related to the PR, please remove.

Reply:
This change allow also to keep consistency in a load/process/save workflow as the name of the low levels components (mesh, materials, ... ) will remains the same instead of being pre and post fixed by some Old-style Radium-specific informations (once apon a time in the Radium life, it seems that pre and postfix of the name - e.g _RO, GEOM_, _MESH_ and so on - where used to identify the type or semantic of these component. Now, we have others informations that allow to identify that and, at least on input/output element, keeping the name unchanged is important in a cooperation workflow.

Reply:
Why not, but maybe in another PR, taking point and vol into account, and remove unnecessary code then (do the `name` variable still in use ?)

Reply:
I do agree.
Let’s go step by step